site2/website/release-notes.md

```diff
@@ -1,6 +1,300 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.10.0
+#### 2022-02-28
+
+### Update notice
+Remove -XX:-ResizePLAB JVM option which degrades performance on JDK11 [#12940](https://github.com/apache/pulsar/pull/12940)
+Enable TCP keepAlive flag on the sockets [#12982](https://github.com/apache/pulsar/pull/12982)
```

Comment:
```suggestion
- Enable TCP keepAlive flag on the sockets [#12982](https://github.com/apache/pulsar/pull/12982)
```

As shown in the preview, the layout is incorrect. Suggest adding `-` before each item

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/50226895/154915937-83b70561-bcc2-4f16-8809-f805d29110ba.png">